osu.Game/Screens/Edit/Compose/Components/Timeline/DifficultyPointPiece.cs

```diff
@@ -145,34 +145,48 @@ protected override void LoadComplete()
 
     internal partial class SliderVelocityInspector : EditorInspector
     {
+        private readonly Bindable<double?> current;
+
+        public SliderVelocityInspector(Bindable<double?> current)
+        {
+            this.current = current;
+        }
+
         [BackgroundDependencyLoader]
         private void load()
         {
             EditorBeatmap.TransactionBegan += updateInspectorText;
             EditorBeatmap.TransactionEnded += updateInspectorText;
+            EditorBeatmap.BeatmapReprocessed += updateInspectorText;
```

Comment:
This is a leak. `SliderVelocityInspector` has a very short lifetime, equal to the lifetime of the popover that contains it. `EditorBeatmap` has a longer lifetime. Thus, because this component doesn't unsubscribe, `EditorBeatmap` will start collecting references to disposed `SliderVelocityInspector`s.

Reply:
Resolved in https://github.com/ppy/osu/pull/23431/commits/2085833a842c023ddb26e85e6fd54a2375867119